ocis/pkg/command/server.go

```diff
@@ -21,7 +20,6 @@ func Server(cfg *config.Config) *cli.Command {
 		},
 		Action: func(c *cli.Context) error {
 			// Prefer the in-memory registry as the default when running in single-binary mode
-			registry.Configure("memory")
```

Comment:
Do we still prefer the `memory` by default or it is a `nats-js-kv` now?
If it is a `nats-js-kv` why do we init the `memory` below?  `r := registry.GetRegistry(registry.Inmemory())`

Reply:
This line was removed. graph will now also default to the `nats-js-kv` registry by default. 

`_test.go` still want to use inmemory registry, hence they get their registry like 
```golang
r := registry.GetRegistry(registry.Inmemory())
```